### ISSUE TITLE
pretrained weights missing

### ISSUE THREAD

**Issue by lokeycookie** [pranoyr/human-attribute-detection-peta#3]
hi, there is no weights file found in the google drive link provided.
Is it possible to upload the pretrained weights?

**Comment by joshuajaistein** [comment]
yes, the pretrained weights is missing @pranoyr can you make it available ?

**Comment by pranoyr** [comment]
Sorry, I don't have the weights with me now.
